test/e2e/basic_workflow_test.go

```diff
@@ -87,6 +93,19 @@ func testServiceCreate(t *testing.T, k kn, serviceName string) {
 	}
 }
 
+func testServiceCreateWithOptions(t *testing.T, k kn, serviceName string, args []string) {
+	command := []string{"service", "create", fmt.Sprintf("%s", serviceName), "--image", KnDefaultTestImage}
+	command = append(command, args...)
+	out, err := k.RunWithOpts(command, runOpts{NoNamespace: false})
+	if err != nil {
+		t.Fatalf(fmt.Sprintf("Error executing 'kn service create' command. Error: %s", err.Error()))
+	}
+
+	if !strings.Contains(out, fmt.Sprintf("Service '%s' successfully created in namespace '%s'.", serviceName, k.namespace)) {
```

Comment:
In other tests, we are testing the output string to contain specific words, for eg:
https://github.com/knative/client/blob/9fb6a436e81d69935a67cc5b245cc634b2ede55b/test/e2e/revision_workflow_test.go#L48

Reply:
Thanks! I changed to use assert here, but it seems that raw `strings.Contains` is used for asserting output text in other e2e tests. Can I follow up to fix them in separate PR?

Reply:
sure!